fontbe/src/features/properties.rs

```diff
@@ -11,12 +12,75 @@ use write_fonts::{
     types::{GlyphId, Tag},
 };
 
+// SAFETY: we can easily verify that neither of these strings contains a nul byte.
+// (this is the only way we can declare these as constants)
+pub const COMMON_SCRIPT: UnicodeShortName =
+    unsafe { UnicodeShortName::from_bytes_unchecked(*b"Zyyy") };
+pub const INHERITED_SCRIPT: UnicodeShortName =
+    unsafe { UnicodeShortName::from_bytes_unchecked(*b"Zinh") };
```

Comment:
Having just gone through considerable delay as a result of unsafe I would very much like to avoid the unsafe.

Perhaps a function with a static OnceCell hidden inside would suffice?

Reply:
this isn't in fontations though, do we have similar concerns with fontc?

Reply:
If this is temporary it's fine to file an issue for it and submit. I wouldn't expect to put fontc in google3 *soon*.